.gitignore

```diff
@@ -0,0 +1,29 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
```

Comment:
believe me you don't need whole this list. Put here only that stuff that you really have in your project.